### DISCUSSION TITLE
Draggable and Resize Dialog.

### DISCUSSION THREAD

**Discussion by Phucmh98** [microsoft/fluentui-blazor#1897] [Ideas]
Currently I'm working with cells in table. Sometimes I need to review the data type so I want to move and resize the Dialog.

**Comment by dvoituron** [comment]
This feature is not included in our Lib: the user cannot resize a dialog box.

**Reply by Phucmh98** [reply]
Hi, Thanks for the reply. I understand that this feature is currently not supported. My idea is to have a resizable and draggable dialog. Will this feature be implemented in the future?

**Reply by vnbaaij** [reply]
No, we have no plans for that. The underlying web component does not support resizing nor dragging

**Reply by Phucmh98** [reply]
Thanks for replying. I will close this topic.
